plugins/community/nipunn1313-mypy-protobuf-grpc/v3.4.0/buf.plugin.yaml

```diff
@@ -1,5 +1,5 @@
 version: v1
-name: buf.build/community/mypy-protobuf-grpc
+name: buf.build/community/nipunn1313-mypy-protobuf-grpc
```

Comment:
Should this be `nipunn1313-mypy-grpc` ?

Reply:
They are both sourced from the same repo nipunn1313-mypy-protobuf - the only difference is the entrypoint to the binary: https://github.com/nipunn1313/mypy-protobuf/blob/7692825cfcf7ef82f2428aaa40e78fb0ba449379/setup.cfg#L25

I thought this would work better to make clear that they're both stemming from a common codebase.